string robot_name
float64 radiation_accumulated
float64 max
float64 min
